app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,11 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
```

Comment:
I don't care much about the memoization, frankly. It smells of premature optimization.

I've got all 💚  now from codeclimate @jessieay and @monfresh -- PTAL